integrations/standalone/tests/pageobjects/AccordionItem.ts

```diff
@@ -19,7 +24,19 @@ export class AccordionItem {
 
   async open() {
     if (!(await this.isOpen())) {
-      await this.locator.click();
+      await this.trigger.locator.click();
     }
   }
+
+  async close() {
+    if (await this.isOpen()) {
+      await this.trigger.locator.click();
+    }
+  }
+
+  async reopen() {
+    await this.close();
+    await this.page.waitForTimeout(200);
+    await this.open();
+  }
```

Comment:
what is this and why is it needed?

Reply:
I use this function to test whether a collapsible is open by default. Reopening an accordion reevaluates the `defaultOpen` property of contained collapsibles. The wait is needed to let the closing animation of the accordion finish. Otherwise, the state of the containing collapsibles is not updated.

Reply:
I do not like this solution, because of the following reasons:
1. What if we ever change the animation time to 400ms? Then this will stop working
2. It will consume test time even if e.g there is no animation

How we can solve this:
1. First of all I would always recommend disabling any animations in the test case, as they consume unnecessary time and also (not sure how to say it) increase the flakyness of a test :). Technically it should work disable the animation by add the reduce motion flag, as the components then will not longer use time for animating: https://github.com/axonivy/form-editor-client/blob/master/playwright/tests/page-objects/form-editor.ts#L22
2. better check the state of the accordion if it is closed, as it will also wait for the accordion to close correctly but consume as much time as needed:
```tsx
async expectClosed() {
  await expect(this.locator).hasAttribute('data-state', 'closed');
}

async reopen() {
  await this.close();
  await expectClosed();
  await this.open();
}
```